package.json

```diff
@@ -63,7 +63,7 @@
     "lint:flow": "babel-node scripts/flow-check.js",
     "plop": "plop",
     "prebootstrap": "yarn",
-    "publish": "lerna publish",
+    "publish": "lerna publish --npm-tag=latest-v1",
```

Comment:
Does this need to be committed? Because we'll just revert it right after, right?

Reply:
this is for `v1` branch, not `master` - we want to be able to push more updates if needed for v1

Reply:
So used to master 🙃 Thanks!